docs/src/components/ComponentClassTable.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import {
+  Table,
+  ComponentClassObject,
+  View,
+  TableRow,
+  TableCell,
+} from '@aws-amplify/ui-react';
+
+export const ComponentClassTable = ({ componentName }) => {
+  const targetClasses = React.useMemo(() => {
+    return Object.values(ComponentClassObject)
```

Comment:
I'm getting an error thrown from line 12: 

![Screen Shot 2022-03-02 at 3 40 42 PM](https://user-images.githubusercontent.com/48109584/156446172-bcf304bc-51b9-42e1-97b4-c189cb83ba08.png)

Reply:
Did you build the react package before running docs? You'll need to do that in order for this component to work because it is expecting that new object